ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -267,6 +267,58 @@ def test_tensorflow_eye(
     )
 
 
+@st.composite
+def elems_reshaped(draw):
+    shape = draw(
+        st.lists(st.integers(min_value=1, max_value=10), min_size=1, max_size=4)
+    )
+    elems_size = np.prod(shape).item()
+    elems = draw(
+        st.lists(
+            st.floats(min_value=-1000, max_value=1000),
+            min_size=elems_size,
+            max_size=elems_size,
+        )
+    )
+    return np.reshape(elems, shape)
```

Comment:
I'm confused as to what this helper even does. From the lines, you are trying to:
- get a shape, i.e. `(1,2,3)`
- get the total size, i.e. `6`
- create a fixed list of floats of size equal to `total size`, i.e. `(1.0,2.0,3.0,4.0,5.0,6.0)`
- reshape it back to original shape, i.e. `[[[1., 2., 3.], [4., 5., 6.]]]`

All of this is entirely possible with `helpers.dtype_and_values` like how it's being used in other tests. Try these arguments
```python
max_value=1000,
max_dim_size=10,
max_num_dims=4,
```

By doing it this way, you can get generate `dtype` as well, and get rid of 
```python
dtype=helpers.get_dtypes("float", full=False),
```

Reply:
I'll check my understanding of the various helpers available and follow through with your suggestion ASAP.

Reply:
Yeah you're understanding of what I'm trying to do is correct, but I think I failed to consider `helpers.dtype_and_values` and what it can do. Like you stated previously, the helper is unnecessary.

Reply:
I resolved the issue with how I wrote the test function according to your suggestions/guidance :)